### DISCUSSION TITLE
Please behave like Opera...

### DISCUSSION THREAD

**Discussion by FraidyMan** [fastaddons/GroupSpeedDial#99] [Q&A]
When I open a new tab in Firefox with the + sign, how do I have the groupsppeddial home page come up? I want it to behave the same a speed dial in Opera....

**Comment by fastaddons** [comment]
My addon should open when you open New tab page.
What page you do see when you open New tab page?
If you have more addons overriding it, the last one installed (or enabled) will be used. So to change it, go to "about:addons" page and disable and enable my addon, that should make it default again.

**Comment by FraidyMan** [comment]
Even with all other add-ons turned off, then FF closed and re-opened, when
I click the + for a new tab, a blank page comes up.  It is likely something
I did barging around in the setup for your add-on.  I had thought about
removing it, but it was a lot of work to set up the groups..

**Reply by fastaddons** [reply]
Checkout Options page / General / "Override "New tab" page with URL" - make sure it's empty.
Also in the Firefox settings (about:preferences#home) what page is set as New tab page.

**Comment by FraidyMan** [comment]
New tabs was set to blank ---Problem solved
Thank you
/\\
